Client.java

```diff
@@ -1,80 +1,56 @@
 import java.util.*;
-import java.io.*;
 
 public class Client{
-   public static void main(String[] args){
+
+   private static final String[] PLAYER_NAMES = {"Bob", "Joe", "Bill", "Kevin"};
+
+//   private static final int MAX_MOVES = 1000;
+
+   private static void nextMoveFor(Player player)
+   {
+      int spin = Board.spin();
+      System.out.println(player.getName() + " spins: " + spin);
+      player.setSpace(Board.getNewLocation(player.getSpace(), spin));
+      System.out.println(player + "\n\n");
+   }
+   public static void main(String[] args)
+   {
       Scanner input = new Scanner(System.in);
       System.out.println("How many players (2 - 4)?");
-      int plrNum = input.nextInt();
-      
-      Player p1 = new Player("Bob");
-      Player p2 = new Player("Joe");
-      Player p3 = new Player("Bill");
-      Player p4 = new Player("Kevin");
-      
-      Board b = new Board();
-      
-      while(p1.getSpace() < 100 && p2.getSpace() < 100 && p3.getSpace() < 100 && p4.getSpace() < 100){

```

Comment:
even if u wanted 2 keep this implementation (w/o an extra variable) doing this is an overhead.
u already have `Player.hasWon()` method and shouldda used it instead of these individual comparisons.
also it would make sense 2 extract this comparison into a meaningfully named method (like `containsWinner(players)`

```java
while(!containsWinner(players))
{
// run game logic
}
```